WooCommerce/Classes/ViewRelated/BottomSheet/ListSelector/BottomSheetListSelectorPresenter.swift

```diff
@@ -6,9 +6,11 @@ import WordPressUI
 final class BottomSheetListSelectorPresenter<Command: BottomSheetListSelectorCommand> {
     private let bottomSheetChildViewController: DrawerPresentableViewController
 
+    /// - Notable Parameters:
+    ///   - onDismiss: Called when the bottom sheet is dismissed. Useful when tapping on each bottom sheet row does not trigger navigation changes.
     init(viewProperties: BottomSheetListSelectorViewProperties,
          command: Command,
-         onDismiss: @escaping (_ selected: Command.Model?) -> Void) {
+         onDismiss: @escaping (_ selected: Command.Model?) -> Void = { _ in }) {
```

Comment:
[nit] I wonder if making it optional and having `nil` as a default value would be more clear for the reader 🤔

Reply:
good call! updated here: https://github.com/woocommerce/woocommerce-ios/pull/2702/commits/56a01e69b97d73e37bd49d066ca23f0d9bc25485